libraries/c_sdk/standard/ble/src/iot_ble_gap.c

```diff
@@ -72,7 +72,7 @@ static const BTUuid_t _serverUUID =
 static IotBleAdvertisementParams_t _scanRespParams =
 {
     .includeTxPower    = true,
-    .nameType          = BTGattAdvNameNone,
+    .name              = { BTGattAdvNameNone,                  0},
```

Comment:
Indentation seems to be off due to uncrustify. Maybe break into multiple lines ?

Reply:
fixed